dashboard/app/controllers/admin_users_controller.rb

```diff
@@ -135,6 +135,33 @@ def user_progress_form
     end
   end
 
+  # GET /admin/user_projects
+  def user_projects_form
+    user_identifier = params[:user_identifier]
+
+    if user_identifier
+      user_identifier.strip!
+      @target_user = User.from_identifier(user_identifier)
+      flash[:alert] = 'User not found' unless @target_user
+    end
```

Comment:
This also seems like a potential for a helper if it's reused elsewhere in the file, though it looks like only one other place